runtime/src/lib.rs

```diff
@@ -384,11 +384,11 @@ parameter_types! {
 pub struct StakingBenchmarkingConfig;
 impl pallet_staking_substrate::BenchmarkingConfig for StakingBenchmarkingConfig {
 	type MaxValidators = ConstU32<1000>;
-	type MaxNominators = ConstU32<1000>;
+	type MaxNominators = ConstU32<16>;
 }
 
 impl pallet_staking_substrate::Config for Runtime {
-	type MaxNominations = ConstU32<1000>;
+	type MaxNominations = ConstU32<16>;
```

Comment:
Note: according to @pLabarta this is the maximum number of validators which each nominator can vote for! Aka how many nominations you can make if you are a nominator.